db/schema.rb

```diff
@@ -1,4 +1,18 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
 ActiveRecord::Schema.define(version: 20160721080417) do
+
+  # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
```

Comment:
Dunno do what you want I remove comments because it was pointed out in the previous pull requests as unnecessary thing.